config/config.toml

```diff
@@ -10,7 +10,7 @@ favicon_url=""
 agreement_url=""
 agreement_version=""
 apple_pay_certificate_url=""
-mixpanel_token=""
+mixpanelToken=""
```

Comment:
you need to change in this file also `src/entryPoints/FeatureFlagUtils.res`

Reply:
hey all the keys here are in snake_case so maintain it